src/plugins/cosmos/components/modals/Staking/views/GetStarted/GetStartedCommon.ts

```diff
@@ -0,0 +1,17 @@
+import { AssetId } from '@shapeshiftoss/caip'
+import { History } from 'history'
+
+export type GetStartedManagerProps = {
+  assetId: AssetId
+}
+
+export type GetStartedRouterProps = {
+  assetId: AssetId
+  stakingRouterHistory: History
+  onClose: () => void
+}
+
+export enum GetStartedManagerRoutes {
+  GetStarted = '/defi/modal/get-started',
+  LearnMore = '/defi/modal/learn-more',
+}
```

Comment:
I'd probably namespace this to cosmos vs. DeFi for now. We're not yet in a state when these are properly reusable as they have ties to Cosmos.